ftplugin/php.vim

```diff
@@ -47,6 +47,8 @@ if !exists('g:phpactorInputListStrategy')
     let g:phpactorInputListStrategy = 'phpactor#input#list#inputlist'
 endif
 
+let g:phpactorUseOpenWindows = get(g:, 'phpactorUseOpenWindows', v:false)
```

Comment:
This syntax looks potentially better, but can we use the same as the above for consistency in this PR?

We are using [vimdoc](https://github.com/google/vimdoc) to generate the docuemntation. Can you add a description of this option as with the other options?

Reply:
(bonus if you can install `vimdoc` and update the help file by running `vimdoc . ` in the project root :) )

Reply:
Vimdoc seems to be unavailable from pip repo but I have installed it manually without any problem. I don't know why vimdoc make this removal

```diff
-NOTE: This help is auto-generated from the VimScript using
-https://github.com/google/vimdoc. See
-https://phpactor.github.io/phpactor/developing.html#vim-help
-
```
and I have to restore it manually.

Please review all these changes for English - also function and variable names ;)

[Rector](https://github.com/rectorphp/rector) for vim is needed ;)

Reply:
> and I have to restore it manually.

I think I added it manually instead of adding it to the `autoload/phpactor.vim` source file ...